examples/Azure_IoT_Central_ESP32/Azure_IoT_Central_ESP32.ino

```diff
@@ -406,14 +411,22 @@ void loop()
 {
   if (WiFi.status() != WL_CONNECTED)
   {
-    azure_iot_stop(&azure_iot);
+    if (azure_iot.state != azure_iot_state_not_initialized)
+      azure_iot_stop(&azure_iot);
+    
     connect_to_wifi();
+    
+    if (!azure_initial_connect)
```

Comment:
Now these two are more serious. This is probably coming because there is possible data corruption when there is wifi disconnection. This seems plausible. We will run a few more tests here.